openshift/ci-operator/generate-ci-config.sh

```diff
@@ -30,7 +30,7 @@ base_images:
   base:
     name: ubi-minimal
     namespace: ocp
-    tag: "8"
+    tag: "9"
```

Comment:
If you wish you can delete this entire script AFAIK it's not used for a long time now.